core/includes/path.inc

```diff
@@ -581,3 +581,145 @@ function backdrop_clear_path_cache($source = NULL) {
   // Clear the backdrop_lookup_path() static cache.
   backdrop_static_reset('backdrop_lookup_path');
 }
+
+/**
+ * Autocomplete callback for paths.
+ *
+ * @param string $string
+ *   The string to find matches for.
+ */
+function path_autocomplete($string = '') {
```

Comment:
Should this be in System Module? It seems curious that the menu link is there but this is in an inc file. And although its a function to find internal paths, it doesnt really have anything to do with core path handling and so on.

Reply:
Yeah, I think I was thinking about this backwards originally, but your suggestion makes sense. Will update.